src/defaults.ts

```diff
@@ -45,7 +45,7 @@ export const OBJECT_DEFAULTS_VECTOR = {
 export const OBJECT_DEFAULTS_TEXT = {
   ...OBJECT_DEFAULTS,
   fill: "black",
-  fontFamily: "Noto Sans Variable",
+  fontFamily: "Verdana",
```

Comment:
Why? Noto Sans is bundled with niimblue.
https://github.com/MultiMote/niimblue/blob/20c1e6c012cfd3032b7fce193e672a368ad0caf1/index.html#L16-L20